src/builders/utils/unimodules.ts

```diff
@@ -19,28 +20,51 @@ interface IPackageInfo {
   dependencies: string[];
 }
 
-export async function resolveExplicitOptIn(workingdir: string, packages?: string[]): Promise<IUnimoduleEntry[]> {
```

Comment:
the changes to `resolveExplicitOptIn` are unrelated and maybe occurred when I ran `yarn lint:fix`